backend/models/lesson.model.ts

```diff
@@ -0,0 +1,36 @@
+import mongoose, { Schema, Document } from "mongoose";
+
```

Comment:
Maybe add a TODO here that says "Replace Object type with Content type"? It'll be helpful once we get those content types defined to find where we need to update.